cmd/starlark/starlark.go

```diff
@@ -121,13 +121,13 @@ func doMain() int {
 			return 1
 		}
 	case flag.NArg() == 0:
-		fromStdin := !term.IsTerminal(int(os.Stdin.Fd()))
-		if !fromStdin {
+		stdinIsTerminal := !term.IsTerminal(int(os.Stdin.Fd()))
```

Comment:
Delete the "!" negations, here and below, so that the name of the variable matches the sense of its meaning.

Reply:
Addressed.